content/applications/websites/website/web_design/building_blocks/inner_content_blocks.rst

```diff
@@ -0,0 +1,39 @@
+=============================
+Inner content building blocks
+=============================
+
+You can use the :guilabel:`Inner content` building blocks, such as the :guilabel:`Button`,
+:guilabel:`Image`, :guilabel:`Video`, or  :ref:`Social Media <inner_content_blocks/social_media>`
+blocks, to enhance your website's view by providing a structure and design for the content available
+on your web pages. Select a block from the :guilabel:`Inner content` section in the
+:guilabel:`Blocks` tab of the website builder, and drag and drop it onto your web page.
+
+.. image:: inner_content_blocks/inner_content_blocks.png
+  :alt: The inner content building blocks available in the website builder.
+
+.. seealso::
+  :doc:`Building blocks <../building_blocks>`
+
+.. _inner_content_blocks/social_media:
+
+Social Media block
+------------------
+
+Displaying your company's social media on your website is a good manner to grow brand awareness and
+improve :doc:`SEO <../../../website/pages/seo>`.
+
+Select the :guilabel:`Social Media` building block from the :guilabel:`Inner content` section and
+drag and drop it anywhere on your website, including in existing building blocks.
```

Comment:
If you don't drag and drop into existing building blocks, it works, but the button goes to the footer -I think there should be a note about that -either here or in the introduction (probably better in the intro -or configuration depending on what you do x))

Reply:
I do not think a note about that would add any value to the documentation. The inner blocks are meant to be added to existing building blocks, as explained in the intro. If users do not drag and drop them to existing blocks (I'm not sure in which case that would be their intention...?), they will see they go to the footer.